src/document.h

```diff
@@ -879,8 +879,7 @@ struct Document {
                         L"<html>\n<head>\n<style>\n"
                         L"body { font-family: sans-serif; }\n"
                         L"table, th, td { border: 1px solid #A0A0A0; border-collapse: collapse;"
-                        L" padding: 3px; vertical-align: top; font-weight: normal;"
-                        L" font-style: normal; font-family: sans-serif; }\n"

```

Comment:
why was this deleted?

Reply:
Because if we do inheritance and we have parent and child share e.g. same font-style that is not the default one (e.g. bold instead of normal), then the code will not add that font-style again for the child as it shall be inherited (this is the goal). But because no font-style is supplied in the child's style attribute then but a default one defined in the style section in the header, that one is prioritized over the inherited one. In order to avoid that situation, do not create this overriding styles in the header.